mobile.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Moible Version</title>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;1,100;1,600&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="mobile.css" />
+</head>
+<body>
+  <div class="home-page">
+    <header>
+      <nav id="navbar">
+        <a href="#" class="logo">My Logo</a>
+        <nav class="menuIcon"><img src="images/menu.png" alt="Menu"></nav>
+        <ul class="internal-link">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section id="home_presentation">
+      <h4 class="header1">I'm Mariane
+        Glad to see you!
+      </h4>
```

Comment:
- [ ] Please make use of the `h1 tag` here, since this is the main header of your page